app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -278,7 +279,8 @@ class BrowserTabViewModel(
         class BrokenSiteFeedback(val data: BrokenSiteData) : Command()
         object DismissFindInPage : Command()
         class ShowFileChooser(val filePathCallback: ValueCallback<Array<Uri>>, val fileChooserParams: WebChromeClient.FileChooserParams) : Command()
-        class HandleExternalAppLink(val appLink: IntentType, val headers: Map<String, String>) : Command()
+        class HandleNonHttpAppLink(val nonHttpAppLink: NonHttpAppLink, val headers: Map<String, String>) : Command()
```

Comment:
I might be missing some context, but what is a `NonHttpAppLink`?

Reply:
For example:

An app link for Maps is: https://maps.google.com
A non-HTTP app link for Maps is: intent://maps.app.goo.gl or market://details?id=com.google.android.apps.maps for the store page.

I don't mind changing the name of the class if you know what those types of links are called? (Or if you can think of a better name). I just know they don't start with "http" 😁